Description: rename async_std feature to std_async
 There is a dependency async-std and a feature async_std both of which
 clashes in names when it reaches debian, hence the feature renaming.
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-05-31
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -83,7 +83,7 @@
 version = "0.2.0"
 
 [features]
-async_std = ["fs"]
+std_async = ["fs"]
 cookie-secure = ["cookies", "cookie/secure"]
 cookies = ["cookie"]
 default = ["fs", "cookie-secure"]
